src/app/facility-form/facility-form.component.scss

```diff
@@ -0,0 +1,78 @@
+.period-group {
+  display: flex;
+  flex-direction: row;
+  padding-right: 15px;
```

Comment:
Are we able to do `em/rem` here instead, or possibly bootstrap `.padding-right-md` on the element(s) in question instead of `padding-right: 15px`.  Same as below, prefer `em` or `rem` instead of `px`?